provider/go.mod

```diff
@@ -5,9 +5,9 @@ go 1.18
 replace github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220824175045-450992f2f5b9
 
 require (
+	github.com/nblockchain/terraform-provider-vultr v0.0.0-20221109104855-966442092901
```

Comment:
@aarani shouldn't this be the only hunk? what are the rest of the changes?

Reply:
Of course not, the SDK gets updated (regenerated) based on the new provider.
Some changes are due to the fact that we are updating to a new version and
some changes are because of our pending PR.

Reply:
then let's mention the URL to commit hash of our repo in the commit msg (after you update it to include upstream PR URL)

Reply:
Done.